crates/bevy_render/src/render_phase/draw_state.rs

```diff
@@ -24,6 +24,9 @@ struct DrawState {
     /// List of vertex buffers by [`BufferId`], offset, and size. See [`DrawState::buffer_slice_key`]
     vertex_buffers: Vec<Option<(BufferId, u64, u64)>>,
     index_buffer: Option<(BufferId, u64, IndexFormat)>,
+
+    /// Stores whether this state is populated or empty for quick state invalidation
+    stores_state: bool,
```

Comment:
Bikeshed: this name confuses me, can we try one of

- `dirty`
- `changed`
- `modified`

instead?

Reply:
I thought of multiple variants when I implemented this, but they do not improve the situation and didn't feel correct for me.

`DrawState` structure stores state.
* `changed`:  It doesn't store if state has changed between different points in time, it stores state information of RenderPass.
* `modified`: The same as with `changed`.
* `dirty`: Stored state doesn't make this structure dirty (It implies something incorrect, not cleaned up). In reality normally this structure should store state, be populated.

That is the reason why I decided to use explicit naming of `store_state` with documentation comment. Maybe `populated` could be used, but explicit name `store_state` made sense for me.

Can change variable name if other reviewers suggest to do so.